### PR TITLE
Allow the user to override the autocomplete results

### PR DESCRIPTION
**Pull request by Jelle-S**

### CODE REVIEW

File: Resources/public/js/countryCodeChange.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ module.exports = {
                             $address.replaceWith(
                                 $(html).find('#' + id)
                             );
-                            $form.find('#' + id + '_countryCode').change(onCountryCodeChange);
+                            var $countryCode = $form.find('#' + id + '_countryCode');
```
Comment (daften) [inline Resources/public/js/countryCodeChange.js:29]: Can't we put this part mostly in the initialization in the js from the bundle?

**Review by daften — COMMENTED** [review]
A check if we can simplify things in the bundle to be used. We also need some changes in the README I'd reckon.

File: Resources/public/js/countryCodeChange.js (hunk 1 above)
Comment (Jelle-S) [reply Resources/public/js/countryCodeChange.js:29]: I'm not sure what you mean?

Comment (daften) [reply Resources/public/js/countryCodeChange.js:29]: Nevermind, I wasn't fully awak I think :D

**Comment by daften** [comment]
When testing locally and creating a new address, I'm getting Missing required property country_code.

```
InvalidArgumentException:
Missing required property country_code.

  at vendor/commerceguys/addressing/src/AddressFormat/AddressFormat.php:125
  at CommerceGuys\Addressing\AddressFormat\AddressFormat->__construct(array('country_code' => '', 'format' => '%givenName %familyName%organization%addressLine1%addressLine2%locality', 'required_fields' => array('addressLine1', 'locality', 'givenName', 'familyName'), 'uppercase_fields' => array('locality'), 'administrative_area_type' => 'province', 'locality_type' => 'city', 'dependent_locality_type' => 'suburb', 'postal_code_type' => 'postal', 'subdivision_depth' => 0))
     (vendor/commerceguys/addressing/src/AddressFormat/AddressFormatRepository.php:24)
  at CommerceGuys\Addressing\AddressFormat\AddressFormatRepository->get('')
     (vendor/daften/addressing-bundle/Service/AddressOutputService.php:140)
  at Daften\Bundle\AddressingBundle\Service\AddressOutputService->getAddressInline(object(AddressEmbeddable))
     (vendor/daften/addressing-bundle/Form/Type/AddressEmbeddableGmapsAutocompleteType.php:78)
  at Daften\Bundle\AddressingBundle\Form\Type\AddressEmbeddableGmapsAutocompleteType->Daften\Bundle\AddressingBundle\Form\Type\{closure}(object(PostSetDataEvent), 'form.post_set_data', object(EventDispatcher))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:264)
```

**Comment by daften** [comment]
When updating an address, the country code is 0 or 1 now.
Postal code isn't updated now it seems.

**Comment by daften** [comment]
@Jelle-S  Provided functional feedback, code looks good so far

**Comment by Jelle-S** [comment]
@daften The errors you're getting is probably because of the changes in the way the form should be created. I'll make a MR against the project as well so you can properly test (so yes, that does mean this introduces breaking changes).

**Pull request merged**